store/sqlstore/upgrade.go

```diff
@@ -367,22 +371,37 @@ func UpgradeDatabaseToVersion471(sqlStore SqlStore) {
 	}
 }
 
+func UpgradeDatabaseToVersion472(sqlStore SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_4_7_1, VERSION_4_7_2) {
+		sqlStore.RemoveIndexIfExists("idx_channels_displayname", "Channels")
+		saveSchemaVersion(sqlStore, VERSION_4_7_2)
+	}
+}
+
 func UpgradeDatabaseToVersion48(sqlStore SqlStore) {
-	if shouldPerformUpgrade(sqlStore, VERSION_4_7_1, VERSION_4_8_0) {
+	if shouldPerformUpgrade(sqlStore, VERSION_4_7_2, VERSION_4_8_0) {
 		saveSchemaVersion(sqlStore, VERSION_4_8_0)
 	}
 }
 
+func UpgradeDatabaseToVersion481(sqlStore SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_4_8_0, VERSION_4_8_1) {
+		sqlStore.RemoveIndexIfExists("idx_channels_displayname", "Channels")
```

Comment:
Strictly speaking, I could remove this line and above (but still migrate the versions), but I figured it's better to preserve the upgrade code between releases to avoid confusion ("Why is this missing?")